apps/api/src/exception-filter.ts

```diff
@@ -96,6 +80,41 @@ export class AllExceptionsFilter implements ExceptionFilter {
       message: `Internal server error, contact support and provide them with the errorId`,
     };
   }
+  private handleCommandValidation(exception: CommandValidationException): ResponseMetadata {
+    const DO_NOT_TRACK_CLASSES = [ProcessBulkTriggerCommand.name];
```

Comment:
why are we choosing not to track this use case?

Reply:
because it's a public API, why do I want to track 4xx errors in it? there is nothing to do with this information besides maybe approach the customer, but if the error is clear enough I see no reason to deal with it, it's not giving us any specific input to a bug in our system, maybe the docs? I just wanted this functionality to exist, the ability to quite down usecases that we don't care about

Reply:
Due to recent error [REF](https://novu-r9.sentry.io/issues/6111320858/?alert_rule_id=15115141&alert_type=issue&notification_uuid=37c1028d-5ad6-442c-885b-874dd9ffb1ac&project=6250907&referrer=slack), I believe one reason to log such errors is that they can help identify edge cases or bugs similar to what occurred last week.

Reply:
i'm removing it, let eveything be logged